frontend/src/components/FacilityBooking/ViewBlock.tsx

```diff
@@ -19,15 +22,24 @@ const ViewBlock = (props: Props) => {
   }, [ref.current])
 
   return (
-    <StyledViewBooking ref={ref} isOccupied={props.entry.type === TimeBlockType.OCCUPIED} blockId={props.entry.id}>
-      {props.entry.type === TimeBlockType.OCCUPIED && (
-        <>
-          {props.entry.ccaName}
-          <br />
-          {props.entry.eventName}
-        </>
+    <>
+      {props.entry.type === TimeBlockType.OCCUPIED ? (
+        <StyledViewBooking
+          ref={ref}
+          isOccupied={true}
+          blockId={props.entry.id}
+          onClick={() => (props.onClickFunction(true), props.setViewBooking(props.entry.booking ?? myBookingsStub[0]))}
```

Comment:
why is it passing true into onclickfunction? should that be done in the corresponding parent onclick instead?; also not sure if using a stub is undefined is a good idea. might want to not trigger setViewBooking if there is no booking instead?

Reply:
Fixed in 374dce79f

Reply:
Booking stub issue fixed in e191997b4